components/api/core/api_impl.bal

```diff
@@ -70,10 +70,8 @@ public function listOrgs(http:Request listOrgsReq, string orgName, int offset, i
             log:printDebug(io:sprintf("Successfully retrieved organization(s) for org name \'%s\'", orgName));
             return buildSuccessResponse(jsonResponse = res);
         } else {
-            string errMsg = "No matching organization found. ";
-            string errDes = io:sprintf("There are no organization(s) for org name \'%s\'", orgName);
-            log:printError(errMsg + errDes);
-            return buildErrorResponse(http:NOT_FOUND_404, constants:API_ERROR_CODE, errMsg, errDes);
+            log:printDebug(io:sprintf("There are no organization(s) for org name \'%s\'", orgName));
+            return buildSuccessResponse();
```

Comment:
I think we should return an empty array if there are no results since the expected result is an array .. WDYT ?